docs/react-v9/contributing/dev-workflow.md

```diff
@@ -1,4 +1,94 @@
+
+That said, some team members use [SourceTree](https://www.sourcetreeapp.com/), [GitHub app](https://desktop.github.com/) or the git integrations available in VS Code.
+
+We don't have an official branch naming policy. However some developers use the following to make their branches easier to find: user/[alias]/you-fancy-branch-name, ie user/johndoe/fix-that-bug
+
+### Building
+
+The inner development loop usually involves these steps:
+
+First, `cd` to the root folder of the repo (usually `fluentui`) and run `yarn` to install dependencies and link packages.
+
+```
+cd fluentui
+yarn // this can take some time
+```
+
+Run `yarn start` and select your start up project.
+
+- @fluentui/public-docsite starts the public v8 documentation site.
+- @fluentui/public-docsite-v9 starts the internal v9 documentation storybook site.
+- Pick a specific component package if you intend to work on a v9 component directly.
+
+Most projects have hot module reloading, so file saves will trigger refreshes.
+
+### Iterating
+
+Make the changes you need, and commit along the way.
+
+```
+git status // Shows all changed files
+git add . // Stages the changed files
+git commit -m "Your brief message." // Makes the commits. You may notice a slight lag during committing as our linters work away.
+git push upstream // Pushes your changes to the central Fluent UI repo.
+```
+
+### Syncing with master
+
+It is strongly recommended that you rebase your branch onto (rather than merging with) with master.
+
+```
+git checkout master // Switches to master
+git pull upstream master // Syncs your local master with the latest version of master at the origin
+git checkout your-fancy-branch // Switches to your branch
+git rebase master // Tacks your commits onto the end of master.
+```
+
+Resolve any conflicts in your editor.
+
+`git push upstream` // Pushes your changes to the central Fluent UI repo.
+
+Creating _draft_ pull requests is often an easy way to keep track of your work without it being actively reviewed by others.
+
+In other cases, such as before checking in or running tests, you may need to run a full build (or build up to a certain package):
+
+- `yarn build` - build everything. It is good the run this weekly, or anytime you start a new project off master.
+- `yarn build --to package-name` - build up to a package.
+  - `yarn build --to @fluentui/react`
+
+You can also `cd` to any package under `packages/*` and run `yarn build` to build individual things, though keep in mind that this may require dependencies to be built first (using `build --to`).
+
+### Making a pull request
+
+Before creating a pull request, be sure to run `yarn change` and provide a high-level description of your change, which will be used in the release notes. When selecting a change type, use this guide:
+
+- Major - Don't do these without expressed agreement from the team. This is a very stable repo.
+- Minor - Adding new API surface area.
+- Patch - No change in API surface area.
```

Comment:
wouldn't mind a confirmation from someone else on the link and list of change types, but I think adding advice on this would be helpful here:

```suggestion
- Patch - No change in API surface area.

After choosing a change type, the description should follow the [semantic commit message format](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716), and be prepended by one of the following: `feat:`, `fix:`, `chore:`, `docs:`, or `refactor:`.
```

Reply:
refactor added 😂🍻

Reply:
@khmakoto I actually saw a `refactor` change file in the 3 most recent change files when I commented, which is why I included it 😂 (but yeah, that was the first one of its type I think I've seen)